helix-term/src/commands/typed.rs

```diff
@@ -1897,6 +1925,13 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
             fun: change_current_directory,
             completer: Some(completers::directory),
         },
+        TypableCommand {
+            name: "change-directory-to-current-file",
+            aliases: &["cdf"],
+            doc: "Change the current working directory to the parent of the current document.",
```

Comment:
I wonder if this should become:
```suggestion
            doc: "Change the current working directory to the parent of the current file.",
```

or if everything else referencing "file" should become "document"...